utils.py

```diff
@@ -0,0 +1,50 @@
+import os
+
+from substrateinterface import SubstrateInterface
+
+
+def get_project_root_dir():
+    return os.path.dirname(os.path.abspath(__file__))
+
+
+# websockets ---------------------------------------
+# dot
+def substratePolkadot():
+    substrate_polkadot = SubstrateInterface(
+        url="wss://rpc.polkadot.io",
```

Comment:
is there something like a checksum we can use to verify we're connected to the right channel/service

Reply:
i really don't know but the domain is reserved and you can find it official docs.
i will debug this.